tests/opentracer/test_tracer_asyncio.py

```diff
@@ -92,3 +98,85 @@ def coro():
         eq_(1, len(traces[0]))
         eq_('coroutine', traces[0][0].name)
 
+
+
+class TestTracerAsyncioCompatibility(AsyncioTestCase):
+    """Ensure the opentracer works in tandem with the ddtracer and asyncio."""
+
+    def setUp(self):
+        super(TestTracerAsyncioCompatibility, self).setUp()
+        self.ot_tracer = get_dummy_asyncio_tracer()
+        self.dd_tracer = self.ot_tracer._dd_tracer
+        self.writer = self.dd_tracer.writer
+
+    @mark_asyncio
+    def test_trace_multiple_coroutines_ot_dd(self):
+        """
+        Ensure we can trace from opentracer to ddtracer across asyncio
+        context switches.
+        """
+        # if multiple coroutines have nested tracing, they must belong
+        # to the same trace
+        @asyncio.coroutine
+        def coro():
+            # another traced coroutine
+            with self.dd_tracer.trace('coroutine_2'):
+                return 42
+
+        with self.ot_tracer.start_active_span('coroutine_1'):
+            value = yield from coro()
+
+        # the coroutine has been called correctly
+        eq_(42, value)
```

Comment:
why using nose here? Always use `pytest` everywhere, otherwise we're adding technical debt.